MovingBlocks/master-server
# Basic migration meta-server to micronauts

### Contains

* Migrated to micronauts framework.
* Removed Jetty and Jax-rs
* Packages moved for common package structure.
* Rewrited tests on micronauts.
* Rewrite docs

Based on #10 

### Tests

1. Setup local run like describe in Readme.md (gradle-way and docker-way)
2. Try every page. should works as previos.
